operations/kubernetes/Manage_Sealed_Secrets.md

```diff
@@ -0,0 +1,181 @@
+
+
+## Manage Sealed Secrets
+
+Procedures for encrypting, decrypting, and modifying Kubernetes sealed secrets.
```

Comment:
It's not obvious to me that this section contains procedures for encrypting, decrypting, and modifying Kubernetes sealed secrets. I would expect there to be subsections "Encrypting sealed secrets", etc.

Note that sealed secrets isn't a Kubernetes thing. "Kubernetes" should be removed from this.

Reply:
Would this section make more sense in the security_and_authentication subdirectory?

Reply:
I had to modify the cray_reds_credentials secret on surtur earlier due to a typo in one of the passwords, I'll write that up into a procedure.

Reply:
@spillerc-hpe do you have that information written up anywhere currently?